README.md

```diff
@@ -3,8 +3,8 @@ PathwaysJob API is an OSS Kubernetes-native API, to deploy ML training and batch
 //ToDo(roshanin) - add intro for Pathways.
 ## Description
 The PathwaysJob is an API that provides an easy way to run JAX workloads using Pathways. It support two modes of deployment. A PathwaysJob instance bundles the Pathways resource manager(RM) AKA Pathways server, the Pathways proxy server and the user workload containers into a single pod called "pathways-head". When the user pod is not provided (headless workloads), the "pathways-head" pod consists of the Pathways RM and the proxy server.
-### Colocate mode
```

Comment:
Thanks Sujeeth,
Please ensure that everything compiles fine by doing `make manifests && make install`

Reply:
Chatted offline that build appears to be successful, but there are some strange things using internal build systems instead. This should be an innocuous enough change.